packages/components/htz-components/src/components/Iframe/Iframe.js

```diff
@@ -0,0 +1,51 @@
+/* global window */
+// @flow
+import React, { useState, useEffect, } from 'react';
+
+import type { Node, } from 'react';
+import type { IframeType, } from '../../flowTypes/IframeType';
+
+type State = {
+  shouldRender: boolean,
+  height: string,
+}
+
+function Iframe({ src, contentName, elementId, ...props }: IframeType): Node {
+  const [ state, setState, ] = useState({
+    shouldRender: false,
+    height: `${props.height}px`,
+  });
+
+  const receiveMessage: Event => void = event => {
+    // $FlowFixMe
+    if (event.data.id === elementId) {
+      setState((prevState: State) => ({
+        ...prevState,
+        // $FlowFixMe
+        height: event.data.height,
+      }));
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener('message', receiveMessage, false);
+    setState((prevState: State) => ({ ...prevState, shouldRender: true, }));
+  });
```

Comment:
You need to return a clean up function that will run when it's time to remove the event listener, also you need to specify an empty dependencies array, which ensures this only runs once, on mount:
```suggestion
    return () => window.removeEventListener('message', receiveMessage)
  }, []);
```